### PR TITLE
apply `when` clauses to accessible view menu, fix some bugs, split code across files

### PR DESCRIPTION
**Pull request by meganrogge**

### CODE REVIEW

**Comment by Tyriar** [comment]
@meganrogge there's a conflict

File: src/vs/workbench/contrib/accessibility/browser/accessibleViewActions.ts [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+}
+registerAction2(AccessibleViewDisableHintAction);
+
+class AccessibleViewFocusToolbarAction extends Action2 {
+	constructor() {
+		super({
+			id: 'editor.action.accessibleViewFocusToolbar',
+			keybinding: {
+				when: accessibleViewIsShown,
+				primary: KeyMod.Alt | KeyCode.F7,
```
Comment (Tyriar) [inline src/vs/workbench/contrib/accessibility/browser/accessibleViewActions.ts:163]: This doesn't seem important enough to have its own keybinding when I'd expect tab or shift-tab to wrap around and focus the actions, does that not happen?

File: src/vs/workbench/contrib/accessibility/browser/accessibleViewActions.ts [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+	run(accessor: ServicesAccessor): void {
+		accessor.get(IAccessibleViewService).disableHint();
+	}
+}
+registerAction2(AccessibleViewDisableHintAction);
+
+class AccessibleViewFocusToolbarAction extends Action2 {
+	constructor() {
+		super({
+			id: 'editor.action.accessibleViewFocusToolbar',
```
Comment (Tyriar) [inline src/vs/workbench/contrib/accessibility/browser/accessibleViewActions.ts:160]: These actions are also present in accessibleView.ts, accident?

**Review by Tyriar — DISMISSED** [review]

File: src/vs/workbench/contrib/accessibility/browser/accessibleViewActions.ts (hunk 1 above)
Comment (meganrogge) [reply src/vs/workbench/contrib/accessibility/browser/accessibleViewActions.ts:163]: yes, that works, I can remove this

**Comment by meganrogge** [comment]
dependency cycle now

```
vs/workbench/contrib/accessibility/browser/accessibleView => 
vs/workbench/contrib/accessibility/browser/accessibleViewActions => 
vs/workbench/contrib/accessibility/browser/accessibleView
```

**Comment by Tyriar** [comment]
I think you can remove the `AccessibleViewAction` reference to fix the cycle. accessibleViewActions should depend on accessibleView and not the other way around, if you needed to share something I'd stick it in another file that they both depend on.

**Comment by meganrogge** [comment]
Yes, I've added an `accessibilityCommands` file to fix this

**Review by Tyriar — APPROVED** [review]

**Pull request merged**